product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/02_connect_on_premises.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Connecting from Networks Outside Azure
+---
+
+
+If your applications are deployed on your on-premises environments, or on other Cloud providers instead Azure, you can set up a secure tunnel between your on premises network and the Azure virtual network. 
+
+There are a few options you can use.
```

Comment:
We only describe one option - let's cut this and simplify. "EDB recommends using Point-to-Site Azure VPN Gateways" or something.